.github/workflows/ci.yml

```
@@ -185,6 +187,7 @@ jobs:
 
       - name: Get secrets from Vault
         id: get-secrets
+        if: ${{ env.IS_FORK != 'true' }}
```

Maybe we should keep it positive
```suggestion
        if: ${{ env.IS_FORK == 'false' }}
```